FileShareService.DesktopClient.Core/Jobs/IJob.cs

```diff
@@ -1,7 +1,14 @@
-﻿namespace UKHO.FileShareService.DesktopClient.Core.Jobs
+﻿using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+
+namespace UKHO.FileShareService.DesktopClient.Core.Jobs
 {
     public interface IJob
     {
         string DisplayName { get; set; }
+        
+        List<string> ErrorMessages { get; set; }
```

Comment:
Please remove the `set` accessor from `ErrorMessages`. Please also change `Validate()` to return void
````
List<string> ErrorMessages { get;  }
void Validate(JToken jsonToken);
````

Reply:
Modified as suggested.